lib/ansible/modules/cloud/packet/packet_device.py

```diff
@@ -91,6 +91,8 @@
   state:
     description:
       - Desired state of the device.
+      - If set to C(present) (the default), the module call will return immediately after the device-creating HTTP request successfully returns.
+      - If set to C(active), the module call will block until all the specified devices are in state active due to the Packet API, which might take few minutes.
```

Comment:
When `state` is `active`, `wait_timeout` is applied and the call won't block more than `wait_timeout`. The documentation of `state`must be updated too.